docs/configuration-reference.md

```diff
@@ -13,6 +13,7 @@ The SR-IOV CNI configures networks through a CNI spec configuration object. In a
 * `mac` (string, optional): MAC address to assign for the VF
 * `spoofchk` (string, optional): turn packet spoof checking on or off for the VF
 * `trust` (string, optional): turn trust setting on or off for the VF
+* `all_multicast` (string, optional): turn allmulticast setting on or off for the VF. Trust setting must be enabled when this setting is on. Note that this option only works for kernel drivers.
```

Comment:
what does it mean kernel drivers ?

Reply:
A: it requires netdev for the VF.

maybe:
"turn allmulticast setting on or off for the VF network device. Trust setting must be enabled when this setting is on."

Reply:
thanks